interface/billing/sl_eob_invoice.php

```diff
@@ -88,7 +88,7 @@ function row_delete($table, $where)
 ?>
 <html>
 <head>
-    <?php Header::setupHeader(['datetime-picker']); ?>
+    <?php Header::setupHeader(['datetime-picker', 'opener', 'no_dialog']); ?>
```

Comment:
it's very interesting that dialog is potentially breaking things. For future, any thing we need to work on a global scale here?

Reply:
what is dialog breaking????

Reply:
oh, I just figured you put the no_dialog there for a reason. This will then not load dialog (which is otherwise always brought in):
https://github.com/openemr/openemr/blob/master/config/config.yaml#L99-L103

Reply:
No reason to load it if don't need so I excluded loading it. The only reason.